### DISCUSSION TITLE
So-status not showing everything?

### DISCUSSION THREAD

**Discussion by Zdroikd** [Security-Onion-Solutions/securityonion#10010] [Unsupported Versions]
I  have just completed the initial setup and can not access the dashboard and it seems like when i run so-status, only 2 things are running? Am i missing something?
![image](https://user-images.githubusercontent.com/129074562/227950785-2ad1c987-ec55-4878-9368-fe85831bb606.png)

**Comment by cm-ops** [comment]
Have you looked at your setup log `/root/sosetup.log` to see if there were any issues with the setup?
